pom.xml

```diff
@@ -49,7 +49,7 @@
   </modules>
 
   <properties>
-    <revision>HEAD-SNAPSHOT</revision>
+    <revision>1.0.0-RC1</revision>
```

Comment:
The revision can always be 'HEAD-SNAPSHOT' cause the pom will never be deployed. And we do it in all the other modules. So, I think, we can go with 'HEAD-SNAPSHOT'